ops/charm.py

```diff
@@ -496,13 +508,15 @@ class RelationMeta:
     :attr:`CharmMeta.relations`.
```

Comment:
I just saw this needs an ', or'
```suggestion
    or :attr:`CharmMeta.relations`.
```